### ISSUE TITLE
FYI, must use dev versions of gbj_htu21 and gbj_twowire

### ISSUE THREAD

**Issue by jskubick** [mrkaleArduinoLib/gbj_htu21#1]
FYI, for anyone who has problems getting gbj_htu21 to build... as of 11/11/2021, you have to use the **dev** branches of *both* gbj_htu21 and gbj_twowire. 

The master branch of gbj_htu21 (latest commit from October 2018, as of today) won't build with either branch of gbj_twowire. 

The dev branch ("6b4a775") commit of gbj_htu21 from December 27, 2018 appears to build and work with the July 2021 ("d130deb") commit of gbi_twowire.

TL/DR: if the latest commit to gbj_htu21's master branch is from 2018... don't use it. Grab the latest commit from the dev branch, and know that you'll probably have to grab gbj_twowire from *its* dev branch, too.
